src/tchap/util/TchapUIFeature.ts

```diff
@@ -28,20 +28,25 @@ export default class TchapUIFeature {
     public static activateClearCacheAndReloadAtVersion4 = true;
 
     /**
-        get list of homeservers where the feature should be activated from config.json
-        example
-         for feature : feature_email_notification
-         add this in config.json
-         {..
-         "tchap":{
+     * Whether the given feature is active on the current user's homeserver.
+     * We get the list of homeservers where the feature should be activated from config.json
+     * Example for feature_email_notification:
+        add this in config.json
+        {..
+            "tchap":{
                 "feature_email_notification": ["dev01.tchap.incubateur.net"]
             }
             ..
         }
     */
     public static isFeatureActiveForHomeserver(feature:string):boolean {
-
         const homeserversWithFeature:[string] = SdkConfig.get("tchap_features")?.[feature] || [];
+
+        // "*" means all servers should have the feature.
+        if (homeserversWithFeature.length === 1 && homeserversWithFeature[0] === "*") {
+            return true;
+        }
```

Comment:
```suggestion
        if (homeserversWithFeature[0] === "*") return true;
```
Je pense que le check du size ne sert pas car s'il est à 0, la valeur sera simplement `undefined` et ne matchera pas la condition :)